### DISCUSSION TITLE
How to create an empty note inside a folder with a defined template

### DISCUSSION THREAD

**Discussion by albertoloscerritos** [SilentVoid13/Templater#1486] [Help] [answered]
Hi. Sorry for the dumb question. But I am faced with the following problem. 
I have a folder called games, and a template for that folder called games_template 

If I use the option `Templater: Create new note from template` and the current file is in the games folder, in the new note the default template for the folder gets replaced with the selected template. So far so good.

Now I want to create a blank template, an empty note called blank_tempalte. This so I can easily create empty notes inside folders with default templates. But when I run `Templater: Create new note from template` and select blank_tempalte, nothing happens, and the new note contains the games_template info. I was expecting the new note to be blank. Why is the blank_tempalte not replacing the games_template. And how can I achieve this? Thanks for reading.

**Comment by Zachatoo** [comment, marked as answer]
There is currently not a way to create a new note from a template, with the template being completely empty, if you have a folder template that includes the folder for the "Default location for new notes" Obsidian setting. If your blank template has anything in it, even a single space character, then the blank template will work.

**Reply by albertoloscerritos** [reply]
Hi thanks for your answer, I really appreciate it. Using a space did the trick :)

Curiously, I can use the completely blank template for "deep" folders:
![image](https://github.com/user-attachments/assets/e90c0e6c-aa59-4fe2-88f8-636402efe52e)
And the blank template will be used, instead of the one defined for /. which is a regular template with properties and more info.

But as you mentioned, I can't do that with the option `Templater: Create new note from template`

Anyway, thanks for the help. Have a nice day 👍
